src/main/java/seedu/mark/logic/parser/ParserUtil.java

```diff
@@ -100,6 +103,104 @@ public static Url parseUrl(String url) throws ParseException {
         return new Url(trimmedUrl);
     }
 
+    /**
```

Comment:
Maybe these methods should be in `TabCommandParser` since they are not going to be used in multiple commands (unlike name, remark which are used in both add and edit).

Reply:
Hmms okay makes sense. I'll move it